.github/CONTRIBUTING.md

```diff
@@ -31,70 +31,93 @@ If you at any point have any questions please read: [I have a question related t
-  - Issues for trackability and history
-  - Why do we do Tech refinement?
-  - Why we do UX refinement?
-  - Why we do UX reviews?
-  - Why do a self-review?
-- General description of the contribution model
+## Introduction to the contribution model
+<!-- Introduction and motivation -->
+Despite Kirby stemming from an internal Bank Data project - it has been decided to open source it, such that others than team Kirby can contribute to it. 
+Allowing for Kirby to be used in multiple projects without team Kirby being the bottleneck. 
+
+If you have a problem, an issue you want resolved, or even got brand new ideas for components. Well. There is no need to sit around and wait for team Kirby to get it implemented. 
+You can contribute yourself! And no matter your role, you can probably contribute with something. May it be a helpful comment or a bug report.
+
+We think that is neat. 
+
+There is still need for managing Kirby however. We need to make sure that the project is moving in the right direction. That submitted code is: high quality, not duplicating anything and consistent with the rest of the codebase.
+To do so a series of steps has been created that an issue must go through. All of these aim at making sure that the process of cooperating from idea to implementation is smooth for contributors. With as little time wasted as possible.
+
+<!-- General description of the contribution model -->
+In broad strokes the process is: submit an issue, get it refined, implement it, get it reviewed. Once reviewed it will automatically be merged to master. 
+
+Actionable checklists for these steps can be found further down in this section. The following highlight some of these steps to explain what they are and why they exist: 
+
+<!-- Why we do UX refinement?-->
+**UX Refinement**: If an issue involves visual changes then UX refinement must take place. This consists of UXers planning how the issue can be solved in a way that aligns with the design system. The `NOT UX refined`-label will be removed from the issue by team Kirby, once it has been done. If a pull-request with visual changes is submitted for an issue that has not been UX refined; there is a good chance that the UX review process may result in a bunch of changes. 
+
+<!--- Why do we do Tech refinement? -->
+**Tech Refinement**: This should happen before any code is written for an issue. Here it is discussed and agreed on how an issue should be implemented the best. How long it takes depends on the scope of the issue. For small issues it might be nothing more than an agreement via a direct message. Other times it is a full-blown meeting with multiple participants. The `NOT Tech refined`-label will be removed from the issue by team Kirby once it has been done. If a pull-request is submitted for an issue that has not been tech refined; changes and problems that could have been tackled from the beginning, might arise from the code-review. 
```

Comment:
Suggestion: Add a sentence about "document the agreed upon tech refinement in a comment on the issue" right before "The `NOT Tech refined`-label will be removed from the issue by team Kirby once it has been done"

Reply:
> The `NOT Tech refined`-label will be removed from the issue by team Kirby once it has been done.

could be 

> Once the issue has been properly tech refined Team Kirby will remove the `NOT Tech refined`-label from the issue

and perhaps emphasise that contributors should not begin to code before this step is finished.

Reply:
Should be solved by new commits ☀️